lib/repositories/auth_repository.dart

```diff
@@ -99,4 +101,10 @@ class AuthRepositoryImpl implements AuthRepository {
     await attempt();
     await fetchUser();
   }
+
+  @override
+  Future<void> resetPassword({required String email}) {
+    final params = AuthResetPasswordParams(email: email);
+    return _authApiService.resetPassword(params: params);
+  }
```

Comment:
these are the backend ticket :\

Reply:
@sleepylee  I know I know, I'm still learning in this process 😢